LuckParser/Models/BossLogic/Deimos.cs

```diff
@@ -139,6 +139,25 @@ public override List<ParseEnum.TrashIDS> GetAdditionalData(CombatReplay replay,
                     replay.AddCircleActor(new CircleActor(true, 0, 180, new Tuple<int, int>(start, end), "rgba(0, 0, 255, 0.3)", new Point3D(-8421.818f, 3091.72949f, -9.818082e8f, 216)));
                 }
             }
+            List<CastLog> annihilate = cls.Where(x => (x.SkillId == 38208) || (x.SkillId == 37929)).ToList();
+            foreach (CastLog c in annihilate)
+            {
+                int start = (int)c.Time;
+                int delay = 1000;
+                int end = start + 2400;
+                int duration = 120;
+                Point3D facing = replay.Rotations.FirstOrDefault(x => x.Time >= start);
```

Comment:
You should always check " != null" after a xxOrDefault. This will crash on logs done between position addition and facing addition to the evtc

Reply:
Ah, I assumed this was handled by the default part, my bad.